src/main/java/com/Timos/TicTacToeInputProducer/Service/TicTacToeInputKafkaProducerService.java

```diff
@@ -0,0 +1,44 @@
+package com.Timos.TicTacToeInputProducer.Service;
+
+import com.Timos.TicTacToeInputProducer.model.InputMessage;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.stereotype.Component;
+
+import java.util.Properties;
+
+@Component
+public class TicTacToeInputKafkaProducerService {
+
+    private final String BOOTSTRAP_SERVERS_ADDRESS = "127.0.0.1:9092";
+    private final String KAFKA_KEY_SERIALIZER = StringSerializer.class.getName();
+    private final String KAFKA_VALUE_SERIALIZER = StringSerializer.class.getName();
+    private final String TOPIC = "TicTacToeTopic";
+
+    public void produceToKafkaTopic(InputMessage inputMessage){
```

Comment:
This is the `TicTacToeInputKafkaProducerService`, does the method really need to be `produceToKafkaTopic`? Would `produce` be ambiguous given the context?